### ISSUE TITLE
Support for AMD

### ISSUE THREAD

**Issue by ErikLarsson82** [bananpermobil/obscen#1]
Hi,

I'm doing a project tonight and I really think Obscen would help.

The project page states :
> Only exports to CommonJS (as of now, AMD is planned).

I really need that AMD support tonight.

/Erik

**Comment by bananpermobil** [comment]
Can you please explain your need in a bit more detail? Do you use npm but not browserify? or are you just copying the source?

**Comment by ErikLarsson82** [comment]
> Do you use npm but not browserify?

Yes, some projects are small enough to not involve browserify, but still warrant the need for Obscen!
